src/core/slave/src/main/resources/slave/config/slave.conf.dist

```diff
@@ -87,6 +87,12 @@ threadedremerge=true
 # Default: [0]
 threadedthreads=0
 
+# The amount of concurrent root collection actions we allow simultaneously.
+# If you have 1 root this will allow 3 concurrent I/O actions
+# If you have 4 roots this will allow 12 concurrent I/O actions to either drive
+# NOTE: If you have ZFS or RAID setups with number of disks and 1 root you will want to change this!
```

Comment:
```suggestion
# NOTE: If you have ZFS or RAID setups with multiple disks and 1 root you will want to increase this.
```
Feels a bit more excplicit, the reader might not have the context if it should be increased or lowered in a RAID/ZFS setup vs multiple standalone disks. Hopefully this edit clarifies it further, as for exact numbers that would be for the user to tweak and find the sweetspot depeding on their array size etc...